Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -1,20 +1,35 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultEditRequest {
+public class BTPayPalVaultEditRequest {
+
+    let editPayPalVaultID: String
```

Comment:
Does this need to be internal for access in the subclass or can it remain private?

Reply:
I see no errors setting it private.